src/Planning/understand-options-tfs-vsts-environments.md

```diff
@@ -1,33 +1,94 @@
+
+![Evolution from datacenters to cloud](_img/understand-options-tfs-vsts-environments/understand-options-tfs-vsts-environments-scenarios.png)
+
+For TFS there are a few deployment options. 
+
+- **Single-server** hosts both the application tier (AT) and the data tier (DT) on a single server. Build, release, test, and proxy servers are optional features and can be deployed on the same or separate servers. A suitable option if you're looking for an easy to use and shareable demo or experimental environment, or simplicity in terms of infrastructure management.
+- **Dual Server (scale up)** starts with the application and the data tiers on separate server. When you are “scaling up,” you are adding resources to a single node in the system, for example, more CPU, memory or disk space.
+- **Multi-server (scale out)** also starts with the application and the data tiers on separate server. However, when you add resources, you add a new node to the system to distribute load and achieve greater capacity. For example, you can add a new server to the application tier to distribute user request load. The scale-out option also adds redundancy and performance by adding additional application and data tier servers. A typical data tier is implemented using SQL Server instances and the application tier by network load balancing (NLB) two or more application tier servers.
+
+[VSTS](https://visualstudio.microsoft.com/team-services/) is the software as a service (SaaS) cloud offering.
+
+## Why should you consider VSTS?
+
+Consider VSTS when you want quick setup, maintenance-free operations, easy collaboration across domains, elastic scale, rock solid security, with access to cloud load testing, cloud build servers, and application insights. It provides a scalable, reliable, backed by a 99.9% SLA, monitored 24x7, and available in local data centers around the world.
+
+![Evolution from datacenters to cloud](_img/understand-options-tfs-vsts-environments/understand-options-tfs-vsts-environments-saas.png)
+
+While both TFS and VSTS provide the same essential services, VSTS provides your organization these added benefits:
+
+- **Always Latest** - No more expensive and complex upgrade cycles to manage your data center. You always have access to the latest version of the service, with optional access to preview features.
+- **CAPEX -> OPEX** - No servers and infrastructure to support, manage, and upgrade. You can move your organizations from resource optimized model, based on capital expenses (CAPEX), to a market optimised business model, based on operational expenses (OPEX). Read [how Microsoft builds its fast and reliable global network](https://azure.microsoft.com/en-gb/blog/how-microsoft-builds-its-fast-and-reliable-global-network/) to understand how Microsoft is continuously innovating and investing their cloud.
```

Comment:
Should the link on building the fast and reliable network be part of the **Scalability** topic? The content of the linked article is more closely related to that topic.

Reply:
Good point. Moved.